stablehlo/reference/Ops.cpp

```diff
@@ -30,13 +33,106 @@ namespace mlir {
 namespace stablehlo {
 namespace {
 
+static Type getF64Type(MLIRContext *context) {
+  return mlir::Builder(context).getF64Type();
+}
+
+static Type getI64Type(MLIRContext *context) {
+  return mlir::Builder(context).getI64Type();
+}
```

Comment:
Also, similarly to the nearby code reviews, let's use the builderless formulations via FloatType::get and IntegerType::get - when builders are not available in the lexical context, they are shorter to type and more idiomatic.